### PR TITLE
Watch later

### PR DESCRIPTION
**Pull request by rishabh486**
pr created for watch later and liked video feature
Screenshot (62)

### CODE REVIEW

File: src/Components/WatchlaterCard/watchlater.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/* :root {
```
Comment (developer-anand) [inline src/Components/WatchlaterCard/watchlater.css:1]: You can remove commented code.

File: src/Pages/SingleVideoPage/SingleVideo.js [hunk 2]
```diff
@@ -51,7 +56,47 @@ function SingleVideo() {
         {video
           .filter((item) => item._id !== videoid)
           .map((filteredVideos) => (
-            <div className="cards-example  card-margin">
+            // <div className="cards-example  card-margin">
```
Comment (developer-anand) [inline src/Pages/SingleVideoPage/SingleVideo.js:59]: Here also.

**Review by developer-anand — APPROVED** [review]
Looks good to Merge.

File: src/Reducers/watch-later.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        },
+      }
+    );
+    console.log(response.data);
+    dispatch({
+      type: "ADD_TO_WATCH_LATER",
+      payload: { watchlater: response.data.watchlater },
+    });
+  } catch (err) {
+    console.log(err.response);
```
Comment (purvasharmadev) [inline src/Reducers/watch-later.js:19]: fix: Remove console. logs, Show error in UI

File: src/Components/WatchlaterCard/watchLater.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    const naviagte=useNavigate()
+    function redirectToSinglePage(id){
+        naviagte(`/single-video/${id}`)
+    }
+    
+  
+  return (<div>
+   
+         {watchlater?.map((videos)=>
+        (
```
Comment (purvasharmadev) [inline src/Components/WatchlaterCard/watchLater.jsx:20]: fix: Add key prop to avoid warnings and unexpected errors

**Review by purvasharmadev — APPROVED** [review]
Good to go after suggested changes. 
code is understandable.

File: src/Components/Liked/like.css [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    width: 100%;
+    height: 40rem;
+  }
+  .cards-example {
+    position: relative;
+    width: 35rem;
+  }
+  /* .primary-button{
+      color:var(--primary-color);
+  } */
```
Comment (itsmebipul00) [inline src/Components/Liked/like.css:83]: Remove all commented code

File: src/Pages/SingleVideoPage/SingleVideo.js [hunk 6]
```diff
@@ -64,26 +109,36 @@ function SingleVideo() {
...
+                      src={like_icon}
+                    />
+                  </div>
                 </div>
                 <div className="top-3">
                   <div class="card-button">
                     <button
                       onClick={() => redirectToSinglePage(filteredVideos._id)}
-                      class="button-container-button primary-button cart"
+                      class="button-container-button primary-button"
```
Comment (itsmebipul00) [inline src/Pages/SingleVideoPage/SingleVideo.js:129]: className**  #fix

**Review by itsmebipul00 — APPROVED** [review]
Good to merge

**Pull request merged**